### ISSUE TITLE
Image objects data unpacking failed

### ISSUE THREAD

**Issue by yogesch** [hardikvasa/google-images-download#327]
For some search terms I get an odd error

```
Evaluating...
list index out of range
Image objects data unpacking failed. Please leave a comment with the above error at https://github.com/hardikvasa/google-images-download/pull/298
```

I think (not too sure) it happens when the results page says "Showing results for ____" and "Search instead for _____". 
Google does this for example when the search terms include a badly misspelled or colloquially used non-standard word.
When I modify the same search term to use only standard words so Google no longer does this ^^ the error also disappears.

**Comment by greg-oz** [comment]
Slightly different error : "Evaluating...
str() takes at most 1 argument (2 given)
Image objects data unpacking failed. Please leave a comment with the above error at https://github.com/hardikvasa/google-images-download/pull/298"

**Comment by Joeclinton1** [comment]
Thank you, for notifying me.

I tried google image search with misspelt queries and I could not reproduce the issue.
Could you please provide the full code/command you used with google_images_download so I can test it myself.

**Comment by shfitz** [comment]
FWIW, I'm getting the same error when I run `python google_images_download.py --keywords "flashlight" --format "jpg" --limit 50`  

If i use `python3`, I receive an error "Could not open URL. Please check your internet connection and/or ssl settings
If you are using proxy, make sure your proxy settings is configured correctly"

**Comment by yogesch** [comment]
> Thank you, for notifying me.
> 
> I tried google image search with misspelt queries and I could not reproduce the issue.
> Could you please provide the full code/command you used with google_images_download so I can test it myself.

Good comment. 

I looked into this a bit further. And the issue is slightly deeper, and not related to the program. The problematic search terms actually lead to 0 results via some google national pages and they show some results other national pages. E.g. in google.co.uk, .com and .co.in, a search term has some results. But in google.nl and .fr it has no results. 

So the confusion was due to me being in one location and downloading on a server in another location. And not realizing that testing in one location doesn't reflect the results from another location. I know ofc that the results will be different but not SO different that some location shows 0 results for the same term. 

It works like it should when I use it from a different location. 

![Screen Shot 2020-09-09 at 01 12 06](https://user-images.githubusercontent.com/18548961/92520797-cdeadb80-f239-11ea-9a33-6dd6b677c3fd.png)

![Screen Shot 2020-09-09 at 01 11 57](https://user-images.githubusercontent.com/18548961/92520939-0985a580-f23a-11ea-892b-c5a8fe61e3d5.png)
